settings/common.py

```diff
@@ -23,6 +23,7 @@
 SITE_VARIABLES = {
```

Comment:
That works well for HEROKU since `devs/admin` don't have access to the system. While using `Ansible` or `puppet` it is easy to separate data from main repo and pull secrets and create a separate file. Environment variables are also ok.